students/mattmaeda/session04/file_lab.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+import os
+
+my_path = os.path.abspath(__file__)
```

Comment:
nice find!

usually, though you want to do stuff in relation to the current working directory --not the dir that that file is in.

Though running code from an IDE or editor can make that challenging...